src/qibolab/transpilers/placer.py

```diff
@@ -215,7 +215,7 @@ def cost(graph, gates_qubits_pairs):
         return len(gates_qubits_pairs) - allowed
 
 
-class Backpropagation(Placer):
+class ReverseTraversalTechnique(Placer):
```

Comment:
```suggestion
class ReverseTraversal(Placer):
```
Maybe? Or do you believe that technique is strictly part of the name?

Reply:
In the paper is Techinque with capital letter! For this reason I thought it is part of the name.

Reply:
In the paper (https://arxiv.org/pdf/1809.02573.pdf)  it is reported as Reverse Transversal Technique (page 7 figure 5) however i noticed that in the caption they use capital letters in every word (i think it is a typo) moreover in page 7 they refer to the tecnique as "Reverse Traversal for Initial Mapping" so maybe technique can be considered redundant.